src/index.js

```diff
@@ -1,15 +1,18 @@
+import { getScores, postScore } from './js/api';
```

Comment:
- Kindly enable the correct [Linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) into your Pull Request.